test/check-ostree

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2015 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+from testlib import *
+import json
+import os
+import sys
+import unittest
+
+INSTALL_RPMS = [
+    "empty-1-0.noarch",
+    "cockpit-shell-wip-0.1.noarch",
+    "cockpit-networkmanager-wip-2.noarch",
```

Comment:
Because that way we can pregenerate these empty RPMs with what will always be a greater and lesser version that what the base tree has installed and test for those changes. All other installed rpms the versions are subject to change depending on the exact version of the tree we happen to run the test against.